src/Domain/Structure/ExcisionSphere.cpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/Structure/ExcisionSphere.hpp"
+
+#include <pup.h>  // IWYU pragma: keep
+#include <pup_stl.h>
+
+#include <ostream>
+
+#include "Utilities/ErrorHandling/Assert.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/StdHelpers.hpp"  // IWYU pragma: keep
+
+template <size_t VolumeDim>
+ExcisionSphere<VolumeDim>::ExcisionSphere(
+    const double radius, const std::array<double, VolumeDim> center) noexcept
```

Comment:
Question: How does this class relate to excision surfaces that are not coordinate-spheres? Does it assume a particular frame in which the excision is a coordinate-sphere, in which case should we template it on the frame?

Reply:
The ExcisionSphere class is written to only hold information for excision surfaces in the grid frame, which are assumed to be coordinate spheres. Information about excision surfaces which are not coordinate spheres, such as spherical harmonic information, will be known by the shape map, so there will be no ExcisionSphere object corresponding to them. So the frame in which the excision is a coordinate sphere is assumed to be the grid frame always, so it is not templated on frame.

But I can will a comment(s) about this to the class so that that is explained :slightly_smiling_face:

Reply:
ok thanks 👍 I would suggest to add pretty much your entire comment here to the docstring, but I'm satisfied with this and there's no need to wait for my approval on this PR.